README.md

```diff
@@ -25,11 +25,25 @@ lang, _ = GetLanguageByContent("foo.m", "<matlab-code>")
 fmt.Println(lang)
 // result: Matlab
 
-lang, _ = GetLanguageByContent("bar.m", "<pbjective-c-code>")
+lang, _ = GetLanguageByContent("bar.m", "<objective-c-code>")
 fmt.Println(lang)
 // result: Objective-C
+
+// all strategies together
+lang := enry.GetLanguage("foo.cpp", "<cpp-code>")
 ```
 
+Developmemt
+-----------
+*enry* re-uses parts of original [linguist](https://github.com/github/linguist) especially data in `languages.yml` to generate internal data structures. In oreder to update to latest upstream run
+
+    make clean code-generate
```

Comment:
Is it better option suggest run make commands than Go ones??

instead `make clean code-generate` you can run `go generate`
instead `make test` you can run `go test -v ./...`

At least for generated files we have generate.go file that allow you to run `go generate`. If the goal is run `make code-generate` instead `go generate` maybe we should remove generate.go file.